### ISSUE TITLE
Add RVN Support X16R Algo

### ISSUE THREAD

**Issue by Dradden** [bonesoul/hypepool#43]
This coin is growing very fast would be awesome to have some support for it!

https://ravencoin.org/wp-content/uploads/2018/01/X16R-Whitepaper-3.pdf

**Comment by ilushka85** [comment]
@Dradden @bonesoul  I added this to coinium I can add here if wanted.

**Comment by Dradden** [comment]
That would be great. Ravencoin is growing at an amazing rate and it uses x16r, not to mention pools for it are having all types of stability issues (mainly yiimp)

**Comment by ilushka85** [comment]
Yes I can add it and have to coinium but need help fixing share difficulty. If someone helps with that I’ll spend time to add this.

On Sat, Mar 3, 2018 at 5:01 PM, Dradden <notifications@github.com> wrote:
That would be great. Ravencoin is growing at an amazing rate and it uses x16r, not to mention pools for it are having all types of stability issues (mainly yiimp)

**Comment by Dradden** [comment]
What exactly is wrong with the share difficulty? I am not a dev but I do have an open line to the x16r devs so i can get you information or questions answered.

**Comment by ilushka85** [comment]
It’s not related to x16r it’s a general thing pools need to calculate and store in order to figure out how much work was done.

On Sat, Mar 3, 2018 at 9:29 PM, Dradden <notifications@github.com> wrote:
What exactly is wrong with the share difficulty? I am not a dev but I do have an open line to the x16r devs so i can get you information or questions answered.
